newdle/client/src/reducers/creation.js

```diff
@@ -99,6 +111,9 @@ export default combineReducers({
   },
   duration: (state = DEFAULT_DURATION, action) => {
     switch (action.type) {
+      case NEWDLE_RECEIVED:
+        // Newdle might be deleted and have no duration (null)
+        return action.newdle.duration || null;
```

Comment:
RFC: As an alternative, we could also modify `fetchNewdle` and return a newdle with:
```javascript
return {duration: null, ...newdle}
```
Not sure what is easier to maintain.

Reply:
i'd keep it in the reducers